packages/trader/src/Stores/Modules/Portfolio/Helpers/details.js

```diff
@@ -62,7 +62,7 @@ export const getDurationUnitText = obj_duration => {
 
 export const getDurationPeriod = contract_info =>
     getDiffDuration(
-        epochToMoment(contract_info.purchase_time || contract_info.date_start),
+        epochToMoment(contract_info.date_start || contract_info.purchase_time),
```

Comment:
:thinking:

Reply:
I guess that's for keeping it in alphabetical order :-)

Reply:
@salar-fs @akmal-binary In forward starting contracts, purchase time is different from start time and the duration is the distance from start time and end time. Without this change, the duration would be incorrect and more than real contract duration.